src/buy-sell.js

```diff
@@ -0,0 +1,25 @@
+
+module.exports = BuySell;
+
+// var buySell = new Blockchain.BuySell(Blockchain.MyWallet.wallet);
+function BuySell (wallet) {
+  this._wallet = wallet;
+
+  // Stop if 2nd password is enabled
```

Comment:
if the point is to stop is second password is enabled why we check for external instead of secondpassword flag?